CHANGELOG.md

```diff
@@ -8,7 +8,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ### Changed
-* Updated `schametype_to_aqnwb` utility script to place the generated source files into a folder hierarchy based on the name of the namespace and schemafile of the neurodata_type (@oruebel, [#224](https://github.com/NeurodataWithoutBorders/aqnwb/pull/224))
+* Enhanced the `schematype_to_aqnwb` utility script:
+    * Generated source files are now placed into a folder hierarchy based on the name of the namespace and schemafile of the neurodata_type (@oruebel, [#224](https://github.com/NeurodataWithoutBorders/aqnwb/pull/224))
+    * Added functionality to optionally create a simple example app that instantiates all generated classes to help test that all generated classes can be compiled (@oruebel, [#225](https://github.com/NeurodataWithoutBorders/aqnwb/pull/225))
+    * Updated generation of header files to ensure proper compilation, e.g.: i) identify an include the headers of all neurodata_types that are being used, ii) fix formatting of comments to avoid nested multi-line comments, iii) fixed issues with incomplete typenames  (@oruebel, [#225](https://github.com/NeurodataWithoutBorders/aqnwb/pull/225))
```

Comment:
```suggestion
    * Updated generation of header files to ensure proper compilation, e.g.: i) identify and include the headers of all neurodata_types that are being used, ii) fix formatting of comments to avoid nested multi-line comments, iii) fixed issues with incomplete typenames  (@oruebel, [#225](https://github.com/NeurodataWithoutBorders/aqnwb/pull/225))
```